www/controllers/curate.js

```diff
@@ -63,13 +67,17 @@ define(['JBrowse/Browser']
             var data = {
                 type:  'correction',
                 value: normalizeFeatures(transcripts)
-            }
+            };
             return http.post('data/tasks/' + id, data).then(function (response) {
                 localStorage.clear();
                 console.log('saved submission');
-            });
-            // what on failure?
-        }
+                },
+                // invoked when request fails (or done button fails)
+                function (response) {
+                    $.confirmDialog(false);
```

Comment:
This isn't really required, right? Because the same, generic error message is shown all the time. And unload handler is always active on the curate page.